### ISSUE TITLE
[CommandButton as menu][Accessibility]: Screen reader doesn't announce expanded/collapsed status for command button as menu and Narrator is unable to navigate to menu items in scan mode when shouldFocusOnMount is set to false.

### ISSUE THREAD

**Issue by raghavendra-kodimala** [OfficeDev/office-ui-fabric-react#8351]
### Environment Information 
 
- __Package version(s)__: (5.122.3) 
- __Browser and OS versions__: (Chrome: 72.0.3626.121 (Official Build) (64-bit), Microsoft Edge: 44.17763.1.0, Microsoft Windows 1809)

#### Actual behavior:
 Screen reader doesn't announce expanded/collapsed states for CommandButton component as menu, and Narrator is unable to focus menu items in scan mode when shouldFocusOnMount is set to false. (If set to false, screen reader announces expanded/collapsed state)
 
#### Expected behavior:
 Screen reader should convey the expanded/collapsed state either shouldFocusOnMount is set to true by default (focus will be on first menu item) or provide access to menu items when, shouldFocusOnMount is set to false.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (No)
 
Requested priority: (High) 
 
Products/sites affected: (if applicable)

**Comment by khmakoto** [comment]
Hi, could you please provide a codepen where the issue is reproing to take a deeper look into it?

**Comment by khmakoto** [comment]
Hi @raghavendra-kodimala, after checking with the Narrator team they tell me that, when a menu is opened the expectation is for the first item to be selected and, because focus goes out of the menu, the state of the menu is no longer read (which is expected behavior).

Regarding access to menu items when `shouldFocusOnMount` is set to `false`, this might be a bug on our side. @dzearing what are your thoughts on this? If you check in [this codepen](https://codepen.io/khmakoto/pen/qvQyLX) you'll see that there is no way to get to the `menuItems` just with keyboard navigation when `shouldFocusOnMount` is set to `false`.

**Comment by dzearing** [comment]
I am confused why down arrow is not moving focus to the first item. That seems like a bug.

If you're in scan mode, which takes over arrowing, I am not sure how to fix that.

https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html

Spec indicates down arrow should expand the menu and move focus to the first item.

**Comment by khmakoto** [comment]
I'll work on a fix for this on non-scan mode scenarios. Thanks!

**Comment by raghavendra-kodimala** [comment]
it should be fixed for scan mode and non-scan mode also

**Comment by khmakoto** [comment]
@raghavendra-kodimala the thing is that the inherent purpose of scan mode is to break out of author's intent when it comes to navigation and dictation. Given that, the fix we provide via arrowing up and down might not work as expected in scan mode.

**Comment by raghavendra-kodimala** [comment]
@khmakoto Navigation to menu items is done through Narrator key (Caps Lock/Insert) + Right/Left arrows normally in scan mode, when expanded. This is expected behavior .
